suites/nautilus/dashboard/sanity_containerized_ceph_dashboard.yaml

```diff
@@ -84,6 +84,12 @@ tests:
             timeout: 300
       desc: Perform rgw tests
 
+   - test:
+      name: Purge ceph dashboard
+      module: purge_dashboard.py
+      polarion-id: CEPH-83573277
+      desc: Purges ceph dashboard
+
```

Comment:
@sunilangadi2 We can use already existing purge something like this instead of creating a new  purge_dashboard.py https://github.com/red-hat-storage/cephci/blob/master/suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml#L185-L192

https://github.com/red-hat-storage/cephci/blob/master/tests/ceph_ansible/purge_cluster.py#L33-L39

Reply:
@Ameenasuhani Yes, know this `purge_cluster.py` to support purge cluster playbook but here lot of other things like 
https://github.com/red-hat-storage/cephci/blob/31922863cb5cf994cd27a6427174d734156cf81c/tests/ceph_ansible/purge_cluster.py#L52-L58
and 
https://github.com/red-hat-storage/cephci/blob/31922863cb5cf994cd27a6427174d734156cf81c/tests/ceph_ansible/purge_cluster.py#L100-L105
there in order to handle purge-cluster playbook which are not needed for purge-dashboard playbook, hence created new `purge_dashboard` module.
So hope you are Ok with this, Please let me know your thoughts?

Reply:
@sunilangadi2 the code you are pointing to is just removing certain packages post purge. what I'm I missing here?

Reply:
@Ameenasuhani that part of code is not needed for dashboard-purge playbook, otherwise it would be messy in that module